modules/context/repo.go

```diff
@@ -518,6 +518,22 @@ func RepoRef() macaron.Handler {
 	return RepoRefByType(RepoRefBranch)
 }
 
+// RefTypeIncludesBranches returns true if ref type can be a branch
+func (rt *RepoRefType) RefTypeIncludesBranches() bool {
```

Comment:
How about just `func (rt RepoRefType)`, since RepoRefType is in fact a `int`.

Reply:
Changed